src/Uno.UWP/UI/Core/CoreWindow.Pointers.Managed.cs

```diff
@@ -24,6 +24,23 @@ public partial class CoreWindow : ICoreWindowEvents
 	public event TypedEventHandler<CoreWindow, KeyEventArgs>? KeyDown;
 	public event TypedEventHandler<CoreWindow, KeyEventArgs>? KeyUp;
 
+#if __SKIA__
+	internal event TypedEventHandler<CoreWindow, WrappedKeyEventArgs>? NativeKeyDownReceived;
+	internal event TypedEventHandler<CoreWindow, WrappedKeyEventArgs>? NativeKeyUpReceived;
+
+	internal class WrappedKeyEventArgs
+	{
+		public KeyEventArgs args { get; set; }
+		public bool IsHandled { get; set; }
+
+		public WrappedKeyEventArgs(KeyEventArgs args, bool isHandled = false)
+		{
+			this.args = args;
+			IsHandled = isHandled;
+		}
+	}
```

Comment:
```suggestion
	internal sealed class WrappedKeyEventArgs
	{
		public KeyEventArgs Args { get; set; }
		public bool IsHandled { get; set; }

		public WrappedKeyEventArgs(KeyEventArgs args)
		{
			Args = args;
		}
	}
```

The rename of `args` -> `Args` will need changes in all places where you access the property.